content/basics/navigating.md

```diff
@@ -4,4 +4,20 @@ date =  2017-10-30T10:09:25-04:00
 weight = 6
 +++
 
-Navigating basics.
+Let's start with a little history. Why does vim use **hjkl** keys to navigate?  
+When Bill Joy created the *Vi* he used the ADM-3A terminal, as you can see the arrows are *hjkl* keys. Then he used the same keys...  
+
+![ADM-3A](/images/ADM-3A.jpg)
+
+Just to remember, to navigate you need to be in *Normal Mode*. We have a suggestion for you, in the section [Useful links](/links)  you'll see a link to **Vim Adventures**. It'll help you to practice/training you muscular memory.
+
+ Key     | Description|
+|--------|------------|
+| **h**  | Left key   |
+| **j**  | Down key   |
+| **k**  | Up key     |
+| **l**  | Right key  |
+
+------------
+
+> just a tip, leave of arrow keys and get used with this keys. You life will be better and you'll be more productive.
```

Comment:
I like the idea of having a *tips* section. Perhaps we can come up with something exclusive here.
For this line:
Quick tip: Try not using the arrow keys and get used to these keys instead. Your life will be better and you'll be more productive.